packages/dd-trace/src/appsec/iast/taint-tracking/constants.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+module.exports = {
+  LOG: 'LOG',
+  REWRITTEN: 'REWRITTEN'
```

Comment:
these constants could be more descriptive:
```suggestion
  LOG_MESSAGE: 'LOG_MESSAGE',
  REWRITTEN_MESSAGE: 'REWRITTEN_MESSAGE'
```